src/harness/testcases/WINNF_FT_S_WDB_testcase.py

```diff
@@ -92,13 +93,26 @@ def generate_WDB_1_default_config(self, filename):
 
     conditionals = [conditionals_b]
 
-    # Create the actual config file
+    pal_db_relative_file_path = os.path.join('testcases', 'testdata', 'pal_db', 'pal_db_record.json')
+
+    # Create the pal_db record file consistent with CBSDs and PAL records.
+    with open(os.path.join(pal_db_relative_file_path), 'w+') as file_handle:
+      file_handle.write(json.dumps(pal_records_a,pal_records_b,indent=2))
```

Comment:
As written, this will write a list of lists to the JSON file. Instead, we want just one list of PAL DB records.

Reply:
We have doubled checked it. This statement 
json.dumps(pal_records_a,pal_records_b,indent=2)
write a list of PAL records.

Reply:
Are you sure that `pal_records_b` is written out? I did an experiment locally and it doesn't seem to be the case.